docs/start-deployment.mdx

```diff
@@ -170,11 +181,11 @@ Finally, in case you want to let people position their inputs on a map, you'll a
 GOOGLE_MAPS_API_KEY=my_google_maps_api_key
 ```
 
-### Other
+### 5.7 Other <DeploymentHelp step={5.7} />
```

Comment:
not sure why this is the odd one out compared to all other 5.x steps, but i tested locally and both work

Reply:
Oops, no reason, just forgot. Will change, risks having floating point precision issues otherwise.